packages/g/gd-sprest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gd-sprest",
+  "filename": "dist/gd-sprest.min.js",
+  "description": "A JavaScript library to interact with the SharePoint REST API.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gunjandatta/sprest"
+  },
+  "keywords": [
+    "sharepoint",
+    "spo",
+    "rest"
+  ],
+  "homepage": "https://dattabase.com/",
+  "npmName": "gd-sprest",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
```suggestion
        "*.@(js|ts)"
```
[Use a more specific glob for just the files needed.](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%2A.%40%28js%7Cts%29&matches=true&tests=%2F%2F%20Imported%20from%20gd-sprest%20NPM%20package&tests=build%2Fhelper%2Fmethods%2FaddContentEditorWebPart.js&tests=build%2Fhelper%2Fmethods%2FaddScriptEditorWebPart.js&tests=build%2Flib%2Fapps.js&tests=build%2Fmapper%2Fcustom%2Faudit.js&tests=build%2Futils%2Fbase.js&tests=build%2Futils%2Fbatch.js&tests=build%2Fhelper%2Fsp%2FcalloutManager.js&tests=clean.js&tests=build%2Flib%2FcontextInfo.js&tests=build%2Fhelper%2Fmethods%2FcreateContentType.js&tests=build%2Fhelper%2Fmethods%2FcreateDocSet.js&tests=build%2Fmapper%2Fdef.js&tests=build%2Fhelper%2Fexecutor.js&tests=build%2Fhelper%2FfieldSchemaXML.js&tests=dist%2Fgd-sprest.js&tests=dist%2Fgd-sprest.min.js&tests=build%2Flib%2Fgraph.js&tests=build%2Fmapper%2Fcustom%2Fgraph.js&tests=build%2Flib%2FgroupService.js&tests=build%2Flib%2FgroupSiteManager.js&tests=build%2Fhelper%2Fmethods%2FhasPermissions.js&tests=build%2Futils%2Fhelper.js&tests=build%2Flib%2FhubSites.js&tests=build%2Flib%2FhubSitesUtility.js&tests=build%2Fhelper%2Findex.js&tests=build%2Fhelper%2Fmethods%2Findex.js&tests=build%2Fhelper%2Fsp%2Findex.js&tests=build%2Findex.js&tests=build%2Flib%2Findex.js&tests=build%2Fmapper%2Fcustom%2Findex.js&tests=build%2Fmapper%2Findex.js&tests=build%2Fsptypes%2Findex.js&tests=build%2Futils%2Findex.js&tests=build%2Fhelper%2Fjslink.js&tests=build%2Flib%2Flist.js&tests=build%2Fhelper%2FlistForm.js&tests=build%2Fhelper%2FlistFormField.js&tests=build%2Futils%2FmethodInfo.js&tests=build%2Fhelper%2Fsp%2FmodalDialog.js&tests=build%2Flib%2Fnavigation.js&tests=build%2Fhelper%2Fsp%2Fnotify.js&tests=build%2Futils%2FoData.js&tests=build%2Fmapper%2Fcustom%2Fold.js&tests=build%2Fhelper%2Fmethods%2Fparse.js&tests=build%2Flib%2FpeopleManager.js&tests=build%2Flib%2FpeoplePicker.js&tests=build%2Fmapper%2Fcustom%2FpeoplePicker.js&tests=build%2Flib%2FprofileLoader.js&tests=build%2Fmapper%2Fcustom%2FpropertyValues.js&tests=build%2Fhelper%2Fmethods%2Frequest.js&tests=build%2Futils%2Frequest.js&tests=build%2Futils%2FrequestType.js&tests=build%2Frest.js&tests=build%2Fhelper%2FribbonLink.js&tests=build%2Fhelper%2FsbLink.js&tests=build%2Flib%2Fsearch.js&tests=build%2Fhelper%2Fmethods%2FsetContentTypeFields.js&tests=build%2Flib%2Fsite.js&tests=build%2Flib%2FsocialFeed.js&tests=build%2Fhelper%2Fsp%2Fsod.js&tests=build%2Fhelper%2FspCfg.js&tests=build%2Fhelper%2FspCfgTypes.js&tests=build%2Fsptypes%2Fsptypes.js&tests=build%2Fhelper%2Fsp%2Fstatus.js&tests=build%2Futils%2FtargetInfo.js&tests=build%2Fhelper%2Ftaxonomy.js&tests=build%2Flib%2FthemeManager.js&tests=build%2Flib%2FuserProfile.js&tests=build%2Flib%2Futility.js&tests=build%2Fmapper%2Fcustom%2Futility.js&tests=build%2Flib%2Fweb.js&tests=build%2Fhelper%2Fwebpart.js&tests=build%2Flib%2FwfInstanceService.js&tests=build%2Flib%2FwfSubscriptionService.js&tests=build%2Futils%2FxhrRequest.js&tests=dts-bundle.json&tests=package.json&tests=LICENSE.md&tests=README.md&tests=.%2F%40types%2Flib%2Fapps.d.ts&tests=.%2F%40types%2Futils%2Fbase.d.ts&tests=.%2F%40types%2Flib%2FcontextInfo.d.ts&tests=.%2F%40types%2Fhelper%2Fexecutor.d.ts&tests=.%2F%40types%2Fhelper%2FfieldSchemaXML.d.ts&tests=dist%2Fgd-sprest.d.ts&tests=.%2F%40types%2Fintellisense%2Fgraph.d.ts&tests=.%2F%40types%2Flib%2Fgraph.d.ts&tests=.%2F%40types%2Flib%2FgroupService.d.ts&tests=.%2F%40types%2Flib%2FgroupSiteManager.d.ts&tests=.%2F%40types%2Futils%2Fhelper.d.ts&tests=.%2F%40types%2Flib%2FhubSites.d.ts&tests=.%2F%40types%2Flib%2FhubSitesUtility.d.ts&tests=.%2F%40types%2Fhelper%2Findex.d.ts&tests=.%2F%40types%2Findex.d.ts&tests=.%2F%40types%2Fintellisense%2Findex.d.ts&tests=.%2F%40types%2Flib%2Findex.d.ts&tests=.%2F%40types%2Fsptypes%2Findex.d.ts&tests=.%2F%40types%2Futils%2Findex.d.ts&tests=.%2F%40types%2Fhelper%2Fjslink.d.ts&tests=.%2F%40types%2Fhelper%2FlinkInfo.d.ts&tests=.%2F%40types%2Flib%2Flist.d.ts&tests=.%2F%40types%2Fhelper%2FlistForm.d.ts&tests=.%2F%40types%2Fhelper%2FlistFormField.d.ts&tests=.%2F%40types%2Fhelper%2Fmethods.d.ts&tests=.%2F%40types%2Flib%2Fnavigation.d.ts&tests=.%2F%40types%2Flib%2FpeopleManager.d.ts&tests=.%2F%40types%2Fintellisense%2FpeoplePicker.d.ts&tests=.%2F%40types%2Flib%2FpeoplePicker.d.ts&tests=.%2F%40types%2Flib%2FprofileLoader.d.ts&tests=.%2F%40types%2Futils%2Frequest.d.ts&tests=.%2F%40types%2Futils%2FrequestType.d.ts&tests=.%2F%40types%2Frest.d.ts&tests=.%2F%40types%2Flib%2Fsearch.d.ts&tests=.%2F%40types%2Flib%2Fsite.d.ts&tests=.%2F%40types%2Flib%2FsocialFeed.d.ts&tests=.%2F%40types%2Fhelper%2Fsp.d.ts&tests=.%2F%40types%2Fhelper%2FspCfg.d.ts&tests=.%2F%40types%2Fhelper%2FspCfgTypes.d.ts&tests=.%2F%40types%2Fsptypes%2Fsptypes.d.ts&tests=.%2F%40types%2Futils%2FtargetInfo.d.ts&tests=.%2F%40types%2Fhelper%2Ftaxonomy.d.ts&tests=.%2F%40types%2Flib%2FthemeManager.d.ts&tests=.%2F%40types%2Flib%2FuserProfile.d.ts&tests=.%2F%40types%2Fintellisense%2Futility.d.ts&tests=.%2F%40types%2Flib%2Futility.d.ts&tests=.%2F%40types%2Flib%2Fweb.d.ts&tests=.%2F%40types%2Fhelper%2Fwebpart.d.ts&tests=.%2F%40types%2Flib%2FwfInstanceService.d.ts&tests=.%2F%40types%2Flib%2FwfSubscriptionService.d.ts&tests=.%2F%40types%2Futils%2FxhrRequest.d.ts&tests=pnpm-lock.yaml)